packages/insert-menu/turbo.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://turbo.build/schema.json",
+  "extends": ["//"],
+  "tasks": {
+    "build": {
+      "dependsOn": ["^build"]
```

Comment:
Since the root `turbo.json` declares that this package should be built, we have to override it here to avoid a recursing dependency that turbo doesn't allow.
This is necessary to begin with due to the root level `.npmrc` setting `link-workspace-packages = deep` which causes the `@sanity/insert-menu` that `sanity` installs to point to the workspace.
Since `sanity` itself isn't a workspace package turbo isn't able to infer that anything that depend on `sanity` should also make sure that `@sanity/insert-menu` is built, we have to manually specify it __somewhere__. Declaring it on the root level ensures that "it just works", and prevents this implementation detail of the monorepo from leaking into lots of affected `turbo.json`'s in the apps folder.